src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -670,6 +671,7 @@ export class InlineChatWidget {
 						insertTextRules: CompletionItemInsertTextRule.InsertAsSnippet,
 						kind: CompletionItemKind.Text,
 						range: new Range(1, 1, 1, 1),
+						command: command.executeImmediately ? { id: AcceptSlashCommand.ID, title: withSlash, arguments: [{ widget: slashCommandActionContext }] } : undefined
```

Comment:
How about using `inlineChat.accept` from [MakeRequestAction](https://github.com/microsoft/vscode/blob/93846caad28583e302b855b3c47c181bd963e37f/src/vs/workbench/contrib/inlineChat/browser/inlineChatActions.ts#L138) instead of defining a new command and context. That would do it for inline chat and I am confident panel chat has a similar command already